.github/workflows/scorecards.yml

```diff
@@ -50,6 +50,6 @@ jobs:
 
       # Upload the results to GitHub's code scanning dashboard.
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@6c089f53dd51dc3fc7e599c3cb5356453a52ca9e # tag=v1.0.26
+        uses: github/codeql-action/upload-sarif@f6e388ebf0efc915c6c5b165b019ee61a6746a38 # tag=v1.0.26
```

Comment:
Note - this tag doesn't seem to be updating as the format seems incorrect. Not a biggie I'll make a `good first issue` for someone to make the tag use the same format that dependabot understands, i.e. `v2.20.1`

Reply:
Good news - our theory was correct, after fixing the tag format dependabot is now updating it, thanks @CaojiamingAlan 👍🏻

Reply:
Good to know